src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int FIRST_COLUMN = 0;
+    private static final int SECOND_COLUMN = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            StringBuilder builder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+            String valueBuilder = builder.toString();
+            String[] counter = valueBuilder.split(System.lineSeparator());
+            int buy = 0;
+            int supply = 0;
+            for (String line: counter) {
+                String[] splitData = line.split("\\W+");
```

Comment:
Please, use constants
 The name of the constant should display this object’s purpose.